src/platform/mock-local-home-platform.ts

```diff
@@ -129,6 +129,10 @@ export class MockLocalHomePlatform implements MockUDPListener {
 
   // Establish fulfillment path using app code
   async onUDPMessage(msg: Buffer, rinfo: RemoteAddressInfo): Promise<void> {
+    if (this.app === undefined) {
```

Comment:
Is the right behavior to silently exit, or to indicate an unexpected problem?

Reply:
Since I'm moving this function to another class in #32, I made the change in #32. (throwing an error and catching it in the test).

Reply:
Since this will be merged first and were a lot of unhandled `undefined` variables, I went ahead and added Error messages.  Will resolve conflicts when merging #32.